### PR TITLE
Bigger one

### PR DESCRIPTION
**Pull request by knil-maloon**
- Added project id on the formatting function for checkboxes as they were displaying wrong.
- Fixed region not showing on concept title page.
- Added new Linked Documents table.
- Modified file function to display size and created a function to display nicely.

### CODE REVIEW

File: classes/ProjectData.php [hunk 1]
```diff
@@ -174,5 +176,37 @@ public static function replaceSymbolsForPDF($sopData){
...
+        // Translate field type, if needed
+        if (isset($fieldTypeTranslator[$fieldType])) {
+            $fieldType = $fieldTypeTranslator[$fieldType];
+        }
+        unset ($Proj);
+        if($fieldType == "checkbox"){
+            return true;
+        }
+        return false;
+    }
```
Comment (paulj3) [inline classes/ProjectData.php:198]: @knil-maloon This is a really good candidate for the framework!

File: classes/ProjectData.php [hunk 2]
```diff
@@ -174,5 +176,37 @@ public static function replaceSymbolsForPDF($sopData){
...
+    public static function formatBytes($bytes, $precision = 2) {
+        $units = array('B', 'KB', 'MB', 'GB', 'TB');
+
+        $bytes = max($bytes, 0);
+        $pow = floor(($bytes ? log($bytes) : 0) / log(1024));
+        $pow = min($pow, count($units) - 1);
+
+        $bytes /= (1 << (10 * $pow));
+
+        return "<span style='font-style:italic;font-size:11px;'>(".round($bytes, $precision) ." ". $units[$pow].")</span>";
```
Comment (paulj3) [inline classes/ProjectData.php:209]: @knil-maloon this too, but may a "utility methods" class or something

File: hub/hub_concept_title.php [hunk 3]
```diff
@@ -572,6 +571,94 @@ function refreshModal(id,link){
         </div>
     </div>
 
+    <div class="panel panel-default" style="margin-bottom: 40px">
+        <div class="panel-heading">
+            <h3 class="panel-title">
+                <a data-toggle="collapse" href="#collapse_publications">Linked Documents</a>
+                <?php
+                $harmonist_perm = ($current_user['harmonist_perms___10'] == 1) ? true : false;
+                if($isAdmin || $concept['contact_link'] == $current_user['record_id'] || $concept['contact2_link'] == $current_user['record_id'] || $harmonist_perm){
```
Comment (paulj3) [inline hub/hub_concept_title.php:580]: This is a lot of ifs...I would consider consolidating into a function for readability...not required though, just a thought

File: classes/ProjectData.php (hunk 2 above)
Comment (knil-maloon) [reply classes/ProjectData.php:209]: I don't think I understand the comment

File: hub/hub_concept_title.php (hunk 3 above)
Comment (knil-maloon) [reply hub/hub_concept_title.php:580]: I might do it once they confirm the logic. I have to ask about that for this.

File: classes/ProjectData.php (hunk 1 above)
Comment (knil-maloon) [reply classes/ProjectData.php:198]: There's a getFieldType function it REDCap I think maybe if I get the time I would add the pid there instead :)

**Pull request merged**